web/src/app/util/TelTextField.tsx

```diff
@@ -46,6 +48,7 @@ export default function TelTextField(
     variables: { number: '+' + phoneNumber },
     requestPolicy: 'cache-first',
     pause: !phoneNumber || props.disabled,
+    context: noSuspense,
```

Comment:
```suggestion
    context: React.useMemo(() => noSuspense, []),
```

Reply:
`useMemo` isn't necessary in these cases because the object is declared a const outside the render function, so the reference never changes.